x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/use_findings_by_resource.ts

```diff
@@ -129,14 +133,20 @@ const createFindingsByResource = (resource: FindingsAggBucket) => {
   if (
     !Array.isArray(resource.cis_sections.buckets) ||
     !Array.isArray(resource.name.buckets) ||
-    !Array.isArray(resource.subtype.buckets)
+    !Array.isArray(resource.subtype.buckets) ||
+    !Array.isArray(resource.cluster_id.buckets) ||
+    !resource.cis_sections.buckets.length ||
+    !resource.name.buckets.length ||
+    !resource.subtype.buckets.length ||
+    !resource.cluster_id.buckets.length
```

Comment:
@ari-aviran this doesn't make sense to me.
We can have empty values IMO and not throw an error on this. wdyt?

if so, i'll fix it on another pr